packages/react-liveness/package.json

```diff
@@ -50,10 +50,10 @@
     "@aws-amplify/ui": "6.0.11",
     "@aws-amplify/ui-react": "6.1.5",
     "@aws-sdk/client-rekognitionstreaming": "3.398.0",
```

Comment:
This package version should also be updated

Reply:
That one didn’t update when I refreshed the lock file

Reply:
Right, the lock file represents the versions installed previously. The source of truth is the actual dependency versions

Reply:
Wait so latest for all of them?